ScnSoft-Education/LeetCodeTraining/LongestSubstring/LongestSubstringSolution.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
```

Comment:
Please add the screenshots with submissions to be able see all submissions and the code, by zooming out the page .